packages/content/lib/index.js

```diff
@@ -138,12 +156,20 @@ module.exports = async function (moduleOptions) {
 
   /* istanbul ignore if */
   if (isSSG) {
+    // Create a hash to fetch the database
+    const dbHash = hash(JSON.stringify(database.items._data)).substr(0, 8)
```

Comment:
Is it okay that not other database changes invalidate hash?

Reply:
Yes since it's only stored in items, right @benjamincanac ?

Reply:
I can use the entire database file to generate the hash if you think it's better but I analyse the file and only the field items._data contains the data !